tests/capture/test_dynamic_shapes.py

```diff
@@ -0,0 +1,160 @@
+# Copyright 2025 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Tests a function for determining abstracted axes and extracting the abstract shapes.
+"""
+# pylint: disable=redefined-outer-name, unused-argument
+
+import pytest
+
+from pennylane.capture import determine_abstracted_axes
+
+pytestmark = pytest.mark.jax
+
+jax = pytest.importorskip("jax")
+jnp = pytest.importorskip("jax.numpy")
+
+
+@pytest.fixture
+def enable_disable():
```

Comment:
```suggestion
def enable_disable_dynamic_shapes():
```
And updating the fixture usage in tests